### PR TITLE
Marketing style change

### PR DESCRIPTION
**Pull request by DeepikaDK001**

### CODE REVIEW

**Review by imranbashir01 — DISMISSED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Models/SolutionFeaturesModel.cs [hunk 1]
```diff
@@ -1,4 +1,8 @@
-﻿using NHSD.GPIT.BuyingCatalogue.EntityFramework.Catalogue.Models;
+﻿using System.Collections.Generic;
+using System.Linq;
+using NHSD.GPIT.BuyingCatalogue.EntityFramework.Catalogue.Models;
+using NHSD.GPIT.BuyingCatalogue.EntityFramework.Ordering.Models;
+using NHSD.GPIT.BuyingCatalogue.Framework.Extensions;
 using NHSD.GPIT.BuyingCatalogue.ServiceContracts.Solutions.Models;
```
Comment (dssherlock) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Models/SolutionFeaturesModel.cs]: Is this change needed?

Comment (DeepikaDK001) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Models/SolutionFeaturesModel.cs]: Thanks for pointing it out. its not needed.. added them accidentally and removing it now.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/SupplierDetails.cshtml [hunk 2]
```diff
@@ -1,53 +1,56 @@
...
-            {
-                <nhs-summary-list>
-                    @if (!string.IsNullOrWhiteSpace(contact.FullName))
-                    {
-                        <nhs-summary-list-row label-text="Contact name">
-                            <p>
-                                @contact.FullName
-                            </p>
-                        </nhs-summary-list-row>
-                    }
```
Comment (dssherlock) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/SupplierDetails.cshtml:32]: Is the extra indentation required or auto-applied? Remove otherwise so it doesnt hide actual code changes

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/Standards.cshtml [hunk 3]
```diff
@@ -106,50 +60,100 @@
...
-    <h2>Capability Specific Standards</h2>
-    <p>These Standards must be fully met due to the Capabilities this Catalogue Solution has mapped to.</p>
+    <nhs-expander label-text="Capability Specific Standards" open="false">
+        <p>These Standards must be fully met due to the Capabilities this Catalogue Solution has mapped to.</p>
 
-    <nhs-details label-text="Standards not fully met">
-        <p>Any Standard that has not been fully met will have a status of ‘in progress’ and the Catalogue Solution will be subject to a Compliance Work-off Plan.</p>
-        <p>This means the supplier has agreed to complete any outstanding work to meet the Standard within an agreed timeframe.</p>
-        <p>Use the ‘view details’ link to find more information about any outstanding Work-off Plans.</p>
-    </nhs-details>
```
Comment (dssherlock) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/Standards.cshtml:120]: See previous indentation comment

**Review by dssherlock — APPROVED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/SupplierDetails.cshtml (hunk 2 above)
Comment (DeepikaDK001) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/SupplierDetails.cshtml:32]: I guess this extra indentation is required to make it more presentable because it has been placed inside the <nhs-expander>.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/Standards.cshtml (hunk 3 above)
Comment (DeepikaDK001) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Solutions/Standards.cshtml:120]: it is also below the <nhs-expander> so it needs this indentation.

**Review by imranbashir01 — APPROVED** [review]
